package.json

```diff
@@ -44,8 +46,11 @@
   },
   "size-limit": [
     {
-      "limit": "650 B",
-      "path": "index.js"
+      "name": "core",
```

Comment:
I think you should pay attention only to this directive.

Reply:
Yeap, we do not need `name` if we have only one limit